enable useHttps for full electron version

sdetweil opened the pull request:
> - Does the pull request solve a **related** issue?
> - If so, can you reference the issue like this `Fixes #<issue_number>`?
Fixes: #2749 
updated  CHANGELOG.md file.

sdetweil commented:
what does the failing codecov/patch test mean?

khassel commented:
this means nothing, looks like adding code lines without adding tests produces this, will be merged anyway ...

sdetweil commented:
how i tested

with the ~/MagicMirror folder as the current folder
i did this 
https://stackoverflow.com/questions/10175812/how-to-generate-a-self-signed-ssl-certificate-using-openssl
search for "You can do that in one command:"
adding -nodes at the end of the listed command

I completed  all the fields except section, which I let default to  blank ([]),
I moved the generated key.pem and cert.pem files to a new folder ~/MagicMirror/keys
I added those pem files to the confg.js, key and cert, and changed useHttps to true
```
    useHttps: true,         // Support HTTPS or not, default "false" will use HTTP
    httpsPrivateKey: "keys/key.pem",     // HTTPS private key path, only require when useHttps is true
    httpsCertificate: "keys/cert.pem",     // HTTPS Certificate path, only require when useHttps is true
```
I started MM, and used chrome to access the pi at https://localhost/:mm_port (as configured in config.js)// note  this is not https://localhost/ (using default port 443)
I clicked advanced at the warning and then proceed on the next page

MM came up...  I was using a local ICS file, which failed to load as it was http:
I changed to https and then added 
                selfSignedCert:true,  // <- this
                calendars: [

to the calendar module config

and restarted MM, and it came up, and was able to access the calendar ics file
